nixops/__main__.py

```diff
@@ -527,6 +527,16 @@
     help="include the physical specification in the evaluation",
 )
 
+subparser = add_subparser(
+    subparsers, "eval", help="eval the given file is nix code in the network expression"
+)
+subparser.set_defaults(op=op_eval)
+subparser.add_argument("code", metavar="CODE", help="code")
+subparser.add_argument(
+    "--json", action="store_true", help="print the option value in JSON format"
```

Comment:
```suggestion
    "--json", action="store_true", help="print the resulting value as an JSON representation of the abstract syntax tree"
```

what does it print without `--json`?